content/en/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

```diff
@@ -34,6 +34,36 @@ etcdctl del "" --prefix
 
 See the [etcd documentation](https://github.com/coreos/etcd/tree/master/etcdctl) for more information.
 
+### CNI configuration cleanup
+
+CNI plugins use the directory `/etc/cni/net.d` to store their configuration.
+The `kubeadm reset` command does not cleanup that directory. Backup the directory if needed.
+Then to remove it execute the following command:
+
+```bash
+sudo rm -rf /etc/cni/net.d
+```
+
+### Network traffic assets cleanup
+
+The `kubeadm reset` command does not clean any iptables, nftables or IPVS assets applied
```

Comment:
changing 'traffic assets' to 'traffic rules', despite IPVS using 'tables', afaik.